Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN apt-get update -y && \
 	apt-get install -y \
 		git \
 		unzip && \
-	curl -Ls -o /usr/bin/k8s-claimer https://storage.googleapis.com/k8s-claimer/git-e4dcc16/k8s-claimer-git-e4dcc16-linux-amd64 && \
+	curl -Ls -o /usr/bin/k8s-claimer https://k8sclaimercli.blob.core.windows.net/cli/k8s-claimer-latest-linux-amd64 && \
```

Comment:
You would still need to trigger a rebuild of hte image to fetch the latest and its done by hand or through a PR to e2e-runner. We can change it back but just felt like this makes it a little easier for us to roll out changes wihtout having to touch a bunch of different repos.